ui/src/components/TrafficPage.tsx

```diff
@@ -50,50 +51,55 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
     const classes = useLayoutStyles();
 
     const [entries, setEntries] = useState([] as any);
+    const [entriesBuffer, setEntriesBuffer] = useState([] as any);
     const [focusedEntryId, setFocusedEntryId] = useState(null);
     const [selectedEntryData, setSelectedEntryData] = useState(null);
     const [connection, setConnection] = useState(ConnectionStatus.Closed);
 
     const [tappingStatus, setTappingStatus] = useState(null);
 
-    const [disableScrollList, setDisableScrollList] = useState(false);
+    const [isSnappedToBottom, setIsSnappedToBottom] = useState(true);
 
-    const [query, setQueryDefault] = useState("");
+    const [query, setQuery] = useState("");
     const [queryBackgroundColor, setQueryBackgroundColor] = useState("#f5f5f5");
+    const [addition, updateQuery] = useState("");
```

Comment:
the convention is to use
```const [param, setParam] = useState("");```
so I think it should be renamed

Reply:
@gadotroee I know the convention.

That state serves a special purpose. It replaces the legacy `updateQuery` method that's used everywhere:

https://github.com/up9inc/mizu/blob/426953c3500a05825eaf72a1878c9d4412348ed2/ui/src/components/TrafficPage.tsx#L87-L94

Therefore we don't want to change its name.